lib/pluginmanager/xpack/pack.rb

```diff
@@ -0,0 +1,82 @@
+# encoding: utf-8
+require "pluginmanager/errors"
+
+module LogStash module PluginManager module Xpack
+  # A object that represent the directory structure
+  # related to the actual gems in the extracted package.
+  #
+  # Example of a valid structure, where `logstash-output-secret` is the actual
+  # plugin to be installed.
+  #.
+  # ├── dependencies
```

Comment:
from what I understand, xpacks will contain a directory called logstash with this structure inside of them, no?